api/src/main/resources/Reference_Application_Diagnoses-11.xml

```diff
@@ -2473,7 +2473,7 @@
   <concept_reference_term concept_reference_term_id="275612" concept_source_id="13" code="62" creator="1" date_created="2014-05-12 00:00:00.0" changed_by="1" retired="false"  uuid="dc5868f6-c78a-3818-8ebd-224da4ba342d"/>
   <concept_reference_term concept_reference_term_id="275613" concept_source_id="13" code="66" creator="1" date_created="2014-05-12 00:00:00.0" changed_by="1" retired="false"  uuid="d0196c86-7bd9-3bdf-931c-2d3d2653bc6b"/>
   <concept_reference_term concept_reference_term_id="275615" concept_source_id="13" code="82" creator="1" date_created="2014-05-12 00:00:00.0" changed_by="1" retired="false"  uuid="c2086097-9ba0-3061-a249-c784d7e0b14b"/>
-  <concept_reference_term concept_reference_term_id="275617" concept_source_id="13" code="122" creator="1" date_created="2014-05-12 00:00:00.0" changed_by="1" retired="false" uuid="fe59f3fc-b1f9-3b79-974b-81dd7698d9cf"/>
+  <concept_reference_term concept_reference_term_id="275617" concept_source_id="13" code="122" creator="1" date_created="2014-05-12 00:00:00.0" changed_by="1" retired="false"  uuid="fe59f3fc-b1f9-3b79-974b-81dd7698d9cf"/>
```

Comment:
@dkayiwa , I have intentionally let these spaces be as they are making the format uniform. Most of the other lines of the similar type have two spaces. These lines had just a single space, so I have added the additional space to make it uniform. Let me know if you feel this shouldn't be

Reply:
Can we use a separate pull request for formatting changes? It becomes hard to review and quickly merge a pull request which has a mixture of actual and formatting changes.